src/BuildingRegistry.Projections.Wfs/Building/BuildingProjections.cs

```diff
@@ -0,0 +1,266 @@
+namespace BuildingRegistry.Projections.Wfs.Building
+{
+    using Be.Vlaanderen.Basisregisters.GrAr.Legacy.Gebouw;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore;
+    using Be.Vlaanderen.Basisregisters.Utilities.HexByteConvertor;
+    using BuildingRegistry.Building.Events;
+    using BuildingRegistry.Building.Events.Crab;
+    using Infrastructure;
+    using NetTopologySuite.Geometries;
+    using NetTopologySuite.IO;
+    using NodaTime;
+
+    public class BuildingProjections : ConnectedProjection<WfsContext>
+    {
+        private static readonly string RealizedStatus = GebouwStatus.Gerealiseerd.ToString();
+        private static readonly string PlannedStatus = GebouwStatus.Gepland.ToString();
+        private static readonly string RetiredStatus = GebouwStatus.Gehistoreerd.ToString();
+        private static readonly string NotRealizedStatus = GebouwStatus.NietGerealiseerd.ToString();
+        private static readonly string UnderConstructionStatus = GebouwStatus.InAanbouw.ToString();
+        private static readonly string MeasuredMethod = GeometrieMethode.IngemetenGRB.ToString();
+        private static readonly string OutlinedMethod = GeometrieMethode.Ingeschetst.ToString();
+
+        private readonly WKBReader _wkbReader;
+
+        public BuildingProjections(WKBReader wkbReader)
+        {
+            _wkbReader = wkbReader;
+
+            When<Envelope<BuildingWasRegistered>>(async (context, message, ct) =>
+            {
+                await context.Buildings.AddAsync(new Building { BuildingId = message.Message.BuildingId });
```

Comment:
pass in `ct` to AddAsync
https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbset-1.addasync?view=efcore-3.1